packages/sdk-components-react/src/time.ws.ts

```diff
@@ -16,14 +16,13 @@ const presetStyle = {
 } satisfies PresetStyle<"time">;
 
 export const meta: WsComponentMeta = {
-  category: "data",
+  category: "localization",
```

Comment:
Single component category doesn't look good. And it is still fits with "data". Why add now?

Reply:
The reason anybody would use Time component is localizing date/time format. They don't need it if they want to just type some date and that has also very little to do with the fact data is remote. Remote nature of data in this case is coinscidence.

IMO user will think: "How do I localize the date" when trying to find that component in the panel

Reply:
I agree single component doesn't look good, lets add currency localization, this is very needed soon for shopify